docs/appendix/common-problems.md

```diff
@@ -119,7 +119,8 @@ If you start up CAVE in Windows and notice the map is showing up only in the bot
 
 Try following these steps to fix your issue:
 
-- Right-click on the CAVE.exe (or shortcut) icon, select Properties
+- Right-click on the CAVE.exe, select Properties
+!!! note "This is not the batch file (CAVE.bat) that gets installed as the CAVE shortcut, the CAVE.exe is located in C:\Users\[your_username]\AppData\Roaming\UCAR Unidata\AWIPS CAVE\CAVE.exe."
```

Comment:
change to "as the CAVE shortcut on your Desktop" maybe

Reply:
How does this look?
![image](https://github.com/Unidata/awips2/assets/21114460/91f2f48d-096f-4faa-bdc2-7dcc3f15a86b)